cmd/nomos/version/version.go

```diff
@@ -136,44 +140,61 @@ func filterConfigs(contexts []string, all map[string]*rest.Config) map[string]*r
 	return cfgs
 }
 
-func lookupVersionAndMode(ctx context.Context, cfg *rest.Config) (string, *bool, error) {
+func getImageVersion(deployment *v1.Deployment) (string, error) {
+	reconcilerImage := strings.Split(deployment.Spec.Template.Spec.Containers[0].Image, ":")
```

Comment:
FYI, it is the reconciler-manager image, not the reconciler.

Reply:
update var name for clarity